Examples/torch/quantization/adaround.py

```diff
@@ -120,7 +120,7 @@ def evaluate(self, model: torch.nn.Module, iterations: int = None, use_cuda: boo
 
 def apply_adaround_and_find_quantized_accuracy(model: torch.nn.Module, evaluator: aimet_common.defs.EvalFunction,
                                                data_loader: torch_data.DataLoader, use_cuda: bool = False,
-                                               logdir: str = '') -> Tuple[torch.nn.Module, float]:
+                                               logdir: str = '') -> float:
```

Comment:
Please remove line 49 `from typing import Tuple` since this import is not being used anymore

Reply:
Right, didn't notice it. Removed.